FCModel/FCModel.m

```diff
@@ -163,6 +163,10 @@ + (instancetype)instanceWithPrimaryKey:(id)primaryKeyValue databaseRowValues:(NS
         // Not in memory yet. Check DB.
         instance = fieldValues ? [[self alloc] initWithFieldValues:fieldValues existsInDatabaseAlready:YES] : [self instanceFromDatabaseWithPrimaryKey:primaryKeyValue];
         if (! instance && create) {
+            if (!g_primaryKeyFieldName[self]) {
```

Comment:
let's just also add the check for `primaryKeyValue` again in case it was returned as nil from the `normalizedPrimaryKeyValue` method